### DISCUSSION TITLE
reconclile invocacation  breaks by adding Owns(&projectcontour.HTTPProxy{}

### DISCUSSION THREAD

**Discussion by setood** [kubernetes-sigs/kubebuilder#2549] [Support] [answered]
my operator works fine if i create controller like this:
``` go
ctrl.NewControllerManagedBy(mgr).
  For(&myoperatorv1.Pool{}).
  Owns(&corev1.Service{}).
  Owns(&corev1.Pod{}).
  // Owns(&projectcontour.HTTPProxy{}).
  Complete(r)
```
and controller func `Reconcile` invoked as expected.
but if uncomment line `Owns(&projectcontour.HTTPProxy{}).`,  func `Reconcile` never called.

`projectcontour` it is import `"github.com/projectcontour/contour/apis/projectcontour/v1"`

**Comment by setood** [comment, marked as answer]
finally the find problem, i forgot add `utilruntime.Must(projectcontourv1.AddToScheme(scheme))` in `main.go`

**Comment by camilamacedo86** [comment]
Closing since it is answered.
